site/content/en/docs/Installation/upgrading.md

```diff
@@ -128,7 +128,63 @@ upgrades.
 1. Start and complete your control plane upgrade(s).
 1. Start and complete your node upgrades.
-1. Scale your Fleets back up and/or recreate your GameServers. 
+1. Scale your Fleets back up and/or recreate your GameServers.
 1. Run any other tests to ensure the Agones installation is still working as expected.
 1. Close your maintenance window.
 7. Congratulations - you have now upgraded to a new version of Kubernetes! 👍
+
+{{% feature publishVersion="1.43.0" %}}
+## SDK Proto Compatibility Guarantees as of Agones v1.43.0
+Our [SDK Server]({{< relref "../guides/Client SDKs/sdk-server.md" >}}) compatibility contract: A game server binary using Beta and Stable SDKs will remain
+compatible with a _newer_ `sdk-server`, within possible deprecation windows:
+- If your game server uses a non-deprecated Stable API, your binary will be compatible for 10 releases (~1y) starting from the SDK version packaged.
+  - For example, if the game server uses non-deprecated stable APIs in the 1.40 SDK, it will be compatible through the 1.50 SDK.
+  - Stable APIs will almost certainly be compatible beyond 10 releases, but 10 releases is guaranteed.
+- If your game server uses a non-deprecated Beta API, your binary will be compatible for 5 releases (~6mo).
+- Alpha SDK Protos are subject to change between releases.
+  - A game server binary using Alpha SDKs may not be compatible with a newer sdk-server.
+  - In Alpha, incompatible changes retaining the same SDK proto message name are allowed.
+  - When we make incompatible Alpha changes, we will document the APIs involved.
+
+## SDK Deprecation Policies as of Agones v1.43.0
+Breaking changes will be called out in upgrade documentation to allow admins to plan their upgrades.
+### Stable Deprecation Policies
+A Stable API may be marked as deprecated in release X and removed from Stable in release X+10.
+### Beta Deprecation Policies
+When a feature graduates from Beta to Stable at release X the API is in SDK proto surfaces from release X to release X+5. The Beta API is marked as deprecated in release X and removed from Beta in release X+5.
+A Beta API may be marked as deprecated in release X and removed from Beta in release X+5 without the API graduating to Stable.
+### Alpha Deprecation Policies
+There is no guaranteed proto compatibility between releases for Alpha SDK protos. When an Alpha API graduates to Beta the API will be deleted from the Alpha proto with no overlapping release.
+An API may be removed from the Alpha proto during any release without graduating to Beta.
+
+## SDK Server APIs and Stability Levels
+
+"Legacy" indicates that this API has been in the SDK Server in a release before we began tracking proto compatibility. \
+The Actions may differ from the [Client SDKs]({{< relref "../guides/Client SDKs/_index.md" >}}) depending on how each Client SDK is implemented.
```

Comment:
I think you can use a `{{< relref "Client SDKs" >}}` here, see https://github.com/googleforgames/agones/blob/db847357163b7ca974553bdf536a2541624ff12b/site/content/en/docs/Advanced/system-diagram.md?plain=1#L49 as an example. (I just noticed how long that line is, didn't even notice in VSCode, woops.)